src/core/agent/services/identifierService.test.ts

```diff
@@ -350,10 +357,68 @@ describe("Single sig service of agent", () => {
   });
 
   // For archive/delete/restore tests
-  test("can delete an archived identifier (identifier and metadata record)", async () => {
-    identifierStorage.getIdentifierMetadata = jest
+  test("should delete all associated linked connections if the identifier is a multi-sig", async () => {
+    identifierStorage.getIdentifierMetadata = jest.fn().mockResolvedValue({
+      ...keriMetadataRecord,
+      isPending: true,
+      signifyOpName: "signifyOpName",
+      isArchived: true,
+    });
+    connections.getMultisigLinkedContacts = jest.fn().mockResolvedValue([
+      {
+        id: "EHxEwa9UAcThqxuxbq56BYMq7YPWYxA63A1nau2AZ-1A",
+        connectionDate: nowISO,
+        label: "",
+        logo: "logoUrl",
+        status: ConnectionStatus.PENDING,
+      },
+    ]);
+    PeerConnection.peerConnection.getConnectingIdentifier = jest
       .fn()
-      .mockResolvedValue(archivedMetadataRecord);
+      .mockReturnValue({ id: archivedMetadataRecord.id, oobi: "oobi" });
+    await identifierService.deleteIdentifier(archivedMetadataRecord.id);
+    expect(connections.deleteConnectionById).toBeCalledTimes(1);
+  });
+
+  test("should delete the local member identifier for that multisig if deleting the multi-sig identifier", async () => {
+    identifierStorage.getIdentifierMetadata
+      .mockReturnValueOnce({
+        ...keriMetadataRecord,
+        isPending: true,
+        signifyOpName: "signifyOpName",
+        multisigManageAid: "manageAid",
+        groupMetadata: undefined,
+        isArchived: true,
+      })
+      .mockReturnValueOnce({
+        ...keriMetadataRecord,
+        isPending: true,
+        multisigManageAid: "manageAid",
+      });
+
+    connections.getMultisigLinkedContacts = jest.fn().mockResolvedValue([
+      {
+        id: "group-id",
+        connectionDate: nowISO,
+        label: "",
+        logo: "logoUrl",
+        status: ConnectionStatus.CONFIRMED,
+      },
+    ]);
+
+    identifierStorage.updateIdentifierMetadata = jest.fn();
+
+    PeerConnection.peerConnection.getConnectingIdentifier = jest
+      .fn()
+      .mockReturnValue({ id: archivedMetadataRecord.id, oobi: "oobi" });
+    await identifierService.deleteIdentifier(archivedMetadataRecord.id);
```

Comment:
same here, you have not called `expect` at all so all you are doing is checking that it doesn't return an error. please check that the connections are deleted and that the member identifier is also deleted